### PR TITLE
chore: translate story and tasks page

### LINKED ISSUE #942: German translations
### Describe the chore

Please translate the following pages from german to english

- https://helpwave.de/product/tasks
- https://helpwave.de/story

### PR DESCRIPTION
**Pull request by MaxSchaefer**
## Which issues does this pull request close?
closes #942

### CODE REVIEW

**Comment by MaxSchaefer** [comment]
@helpwave/medical requesting proofreading

File: landing-page/components/sections/tasks/StartSection.tsx [hunk 1]
```diff
@@ -2,8 +2,30 @@ import { tw, tx } from '@helpwave/common/twind'
...
+}
+
+const defaultStartSectionTranslation: Record<Languages, StartSectionTranslation> = {
+  en: {
+    title: 'The first open-source team management platform for healthcare workers',
+    text: '\\helpwave \\primary{tasks} increases the productivity in your clinical team. Ditch your print-out ward list and coordinate your work and the work of your team in a modern kanban interface. \\helpwave \\primary{tasks} has been inspired by modern industry work flows and best practices from lean project management and applies them to the clinical environment.'
+  },
+  de: {
+    title: 'Die erste Open-Source-Team-Management-Plattform für Mitarbeitende im Gesundheitswesen',
+    text: '\\helpwave \\primary{tasks} erhöht die Produktivität in deinem klinischen Team. Lass die ausgedruckten Stationslisten hinter dir und koordiniere deine Arbeit und die deines Teams mit einer modernen Kanban-Schnittstelle. \\helpwave \\primary{tasks} wurde von modernen Arbeitsabläufen und Best.Practices aus dem Lean-Projektmanagement inspiriert und wendet diese Methodiken auf die klinische Umgebung an.'
```
Comment (DasProffi) [inline landing-page/components/sections/tasks/StartSection.tsx:22]:
```suggestion
    text: '\\helpwave \\primary{tasks} erhöht die Produktivität in deinem klinischen Team. Lass die ausgedruckten Stationslisten hinter dir und koordiniere deine Arbeit und die deines Teams mit einer modernen Kanban-Schnittstelle. \\helpwave \\primary{tasks} wurde von modernen Arbeitsabläufen und Best Practices aus dem Lean-Projektmanagement inspiriert und wendet diese Methodiken auf die klinische Umgebung an.'
```

File: landing-page/components/sections/tasks/ReachoutSection.tsx [hunk 2]
```diff
@@ -1,21 +1,70 @@
...
+      \\helpwave \\primary{tasks} has been inspired by modern industry work flows and best practices from lean project management and applies them to the clinical environment.
+      Keeping track of tasks and workflows in a clinical environment leads to better patient care and a more efficient use of resources.
+      \\helpwave \\primary{tasks} can increase worker satisfaction and patient safety. Use it as a secure basis for your billing processes.
+    `
+  },
+  de: {
+    medicalHeroTitle: 'Du bist ein medical hero?\n',
+    medicalHeroSubtitle: '\\helpwave \\primary{tasks} zielt darauf ab, dich bei deiner täglichen Arbeit zu unterstützen!',
+    medicalHeroText: `
+      \\helpwave \\primary{tasks} ist eine moderne Kanban-Schnittstelle, die dabei hilft, deine Arbeit die deines Teams zu koordinieren. Notiere dir deine Aufgaben für deinen Patienten für dich selbst und für dein Team.
```
Comment (DasProffi) [inline landing-page/components/sections/tasks/ReachoutSection.tsx:36]:
```suggestion
      \\helpwave \\primary{tasks} ist ein modernes Kanban-Interface, das dabei hilft, deine Arbeit und die Arbeit deines Teams zu koordinieren. Notiere dir deine Aufgaben für deine Patienten, für dich selbst und für dein Team.
```

File: landing-page/components/sections/tasks/ReachoutSection.tsx [hunk 3]
```diff
@@ -1,21 +1,70 @@
...
+      Standardisiere deine Workflows, indem du sie als Vorlage speicherst und mit deinen neuen Kollegen für ein einfaches Onboarding teilst. Erhalte Benachrichtigungen über bevorstehende Aufgaben und nutzen unsere verschiedenen Dashboards,
```
Comment (DasProffi) [inline landing-page/components/sections/tasks/ReachoutSection.tsx:37]:
```suggestion
      Standardisiere deine Workflows, indem du sie als Vorlage speicherst und mit neuen Kollegen für ein einfaches Onboarding teilst. Erhalte Benachrichtigungen über bevorstehende Aufgaben und nutzen unsere verschiedenen Dashboards,
```

File: landing-page/components/sections/tasks/ReachoutSection.tsx [hunk 4]
```diff
@@ -1,21 +1,70 @@
...
+      um Patienten und deren Behandlungsverlauf zu verfolgen. \\helpwave \\primary{tasks} ist die erste Open-Source-Team-Management-Plattform für Mitarbeiter im Gesundheitswesen.
+    `,
+    hospitalManagerTitle: 'Du arbeitest in der Krankenhausverwaltung?',
+    hospitalManagerSubtitle: 'Wie \\helpwave \\primary{tasks} deine Probleme auch hier lösen kann!',
+    hospitalManagerText: `
+      \\helpwave \\primary{tasks} wurde von modernen Arbeitsabläufen und Best.Practices aus dem Lean-Projektmanagement inspiriert und wendet diese Methodiken auf die klinische Umgebung an.
```
Comment (DasProffi) [inline landing-page/components/sections/tasks/ReachoutSection.tsx:43]:
```suggestion
      \\helpwave \\primary{tasks} wurde von modernen Arbeitsabläufen und Best Practices aus dem Lean-Projektmanagement inspiriert und wendet diese Methodiken auf die klinische Umgebung an.
```

File: landing-page/components/sections/tasks/ReachoutSection.tsx [hunk 5]
```diff
@@ -1,21 +1,70 @@
...
+      Die Verfolgung von Aufgaben und Arbeitsabläufen in einem klinischen Umfeld führt zu einer besseren Patientenversorgung und einer effizienteren Nutzung von Ressourcen.
+      \\helpwave \\primary{tasks} kann die Zufriedenheit der Mitarbeiter und die Patientensicherheit erhöhen. Du kannst \\helpwave \\primary{tasks} ausch als sichere Grundlage für deine Abrechnungsprozesse verwenden.
```
Comment (DasProffi) [inline landing-page/components/sections/tasks/ReachoutSection.tsx:45]:
```suggestion
      \\helpwave \\primary{tasks} kann die Zufriedenheit der Mitarbeiter und die Patientensicherheit erhöhen. Du kannst \\helpwave \\primary{tasks} auch als sichere Grundlage für deine Abrechnungsprozesse verwenden.
```

File: landing-page/components/sections/tasks/FeatureSection.tsx [hunk 6]
```diff
@@ -1,8 +1,40 @@
...
+    collaborateTitle: 'Collaborate',
+    collaborateText: 'You and your team synchronized on the same heartbeat.',
+    patientsTitle: 'Patients',
+    patientsText: 'Modern dashboards for your patients. Keep track of their progress and tasks.'
+  },
+  de: {
+    taskTemplatesTitle: 'Aufgabenvorlagen',
+    taskTemplatesText: 'Speichere sich wiederholende Aufgaben als Vorlagen. Standardisiere deine Arbeitsabläufe und teile sie mit deinem Team.',
+    collaborateTitle: 'Zusammenarbeiten',
+    collaborateText: 'Du und dein Team arbeiten auf den Herzschlag um Herzschlag synchronisiert zusammen.',
```
Comment (DasProffi) [inline landing-page/components/sections/tasks/FeatureSection.tsx:30]: Maybe we should not translate it literally
```suggestion
    collaborateText: 'Du und dein Team arbeiten auf den Herzschlag synchronisiert zusammen.',
```

**Review by DasProffi — APPROVED** [review]

**Comment by DasProffi** [comment]
We maybe want to change the layout on the tasks product page as certain texts are not aligned properly like the "Zusammenarbeit" title. Furthermore, some longer texts which are center aligned are hard to read, because the start of a line is different for every line

**Pull request merged**